honeycombio/resource_marker_settings.go

```diff
@@ -0,0 +1,105 @@
+package honeycombio
```

Comment:
This is a nit, but I'd like to see the name of this file match the resource (`resource_market_setting.go` without the final "s")